splinter/browser.py

```diff
@@ -78,12 +76,24 @@ def get_driver(driver, retry_count=3, *args, **kwargs):
 
     """
     err = None
+    handle_excs = ()
```

Comment:
@blueyed I think a better approach here is to do the try/except for the exception classes outside the browser init loop:

```
driver_exceptions = (IOError, HTTPException, MaxRetryError)
try:
    from selenium.common.exceptions import WebDriverException
    driver_exceptions += (WebDriverException,)
except ImportError as e:
    logger.debug(f"Import Warning: {e}")
```

and then:
```
except driver_exceptions as e:
    err = e
```

Reply:
Makes sense, pushed a fixup (which should get squash-merged).

However, please note that my initial idea was to delay the import until it is necessary (in case of an error), but I am fine with having it being tried always.